Makefile_gnu

```diff
@@ -32,7 +32,7 @@ endif
 ifneq (,$(findstring clang,$(shell '$(CXX)' -v 2>&1)))
 LDLIBS += -lc++
 else
-LDLIBS += -lstdc++
+LDLIBS += -lstdc++ -lmpi_cxx
```

Comment:
I have confirmed that this indeed fails without this flag on my Arch Linux system as well